### PR TITLE
last update of BEOL ontologies

### PR DESCRIPTION
**Pull request by SepidehAlassi**

### CODE REVIEW

File: webapi/_test_data/ontologies/beol-onto.ttl [hunk 1]
```diff
@@ -1358,8 +1580,6 @@
                 rdfs:label  "Manuskripteintrag in"@de ,
                             "Manuscript entry in"@en ;
 
-                knora-base:subjectClassConstraint :manuscriptEntry ;
```
Comment (tobiasschweizer) [inline webapi/_test_data/ontologies/beol-onto.ttl]: Why did you remove the subject class constraint?

File: webapi/_test_data/ontologies/beol-onto.ttl [hunk 2]
```diff
@@ -1372,8 +1592,6 @@
 
                 knora-base:objectClassConstraint knora-base:LinkValue ;
 
-                knora-base:subjectClassConstraint :manuscriptEntry ;
-
```
Comment (tobiasschweizer) [inline webapi/_test_data/ontologies/beol-onto.ttl]: same question as above

File: webapi/_test_data/ontologies/beol-onto.ttl (hunk 1 above)
Comment (SepidehAlassi) [reply webapi/_test_data/ontologies/beol-onto.ttl]: because this property is being used by the biblio:VPArticle resource as well.

File: webapi/_test_data/ontologies/beol-onto.ttl (hunk 2 above)
Comment (SepidehAlassi) [reply webapi/_test_data/ontologies/beol-onto.ttl]: see my answer above

File: webapi/scripts/fuseki-load-test-data.sh [hunk 3]
```diff
@@ -18,4 +18,5 @@ curl -F filedata=@../_test_data/ontologies/beol-onto.ttl http://localhost:3030/k
 curl -F filedata=@../_test_data/ontologies/biblio-onto.ttl http://localhost:3030/knora-test/data?graph=http://www.knora.org/ontology/0801/biblio > /dev/null
 curl -F filedata=@../_test_data/all_data/biblio-data.ttl http://localhost:3030/knora-test/data?graph=http://www.knora.org/data/biblio > /dev/null
 curl -F filedata=@../_test_data/all_data/beol-data.ttl http://localhost:3030/knora-test/data?graph=http://www.knora.org/data/beol > /dev/null
-
+curl -F filedata=@../_test_data/ontologies/newton-onto.ttl http://localhost:3030/knora-test/data?graph=http://www.knora.org/ontology/0801/newton > /dev/null
+curl -F filedata=@../_test_data/ontologies/leibniz-onto.ttl http://localhost:3030/knora-test/data?graph=http://www.knora.org/ontology/0801/leibniz > /dev/null
```
Comment (tobiasschweizer) [inline webapi/scripts/fuseki-load-test-data.sh]: ending newline is missing

File: webapi/_test_data/ontologies/beol-onto.ttl [hunk 4]
```diff
@@ -2151,13 +2369,13 @@
         [
             rdf:type owl:Restriction ;
             owl:onProperty :manuscriptEntryOf ;
-            owl:cardinality "1"^^xsd:nonNegativeInteger ;
+            owl:minCardinality "0"^^xsd:nonNegativeInteger ;
```
Comment (tobiasschweizer) [inline webapi/_test_data/ontologies/beol-onto.ttl]: once we have imported, everything, this can be set back to 1

**Comment by tobiasschweizer** [comment]
@SepidehAlassi Please merge this PR as soon as the tests have passed.

**Comment by SepidehAlassi** [comment]
@tobiasschweizer thanks for merging :D
